### DISCUSSION TITLE
Monatliche Aggregation der Sonnenenergie in Charts

### DISCUSSION THREAD

**Discussion by Brezentrager** [roe-dl/weewx-GTS#21] [Q&A] [answered]
Hallo,
ich versuche gerade ein Belchertown-Chart zu basteln, in dem die monatlichen Summen der Sonnenenergie dargestellt werden. Also analog zu diese Chart von woellsdorf-wetter:
<img width="595" alt="Bildschirmfoto 2023-03-08 um 21 48 54" src="https://user-images.githubusercontent.com/8404725/223847078-0b10c28a-612d-4d6b-8fcf-24ef72ab81d4.png">

Leider gelingt es mir nicht mit dem üblichen "xAxis_grouping = month"

Könntest Du mir bitte Deine Chart-Definition aus der graphs.conf zeigen?

Danke!

**Comment by roe-dl** [comment, marked as answer]
```
    [[SonnenenergieVorjahr]]
        title = "Sonnenenergie (monatlich gesamt, Vorjahr)"
        time_length = years_ago
        time_ago = 1
        aggregate_type = energy_integral
        aggregate_interval = 2629800
        start_at_midnight = true
        #start_at_beginning_of_month = true
        yAxis_label = Energie
        exporting = 1
        [[[radiation]]]
            zIndex = 1
            type = column
            color = "#ffc83f"
            unit = kilowatt_hour_per_meter_squared
```

P.S.: Man kann übrigens über das Sandwich-Menü (die drei Striche) oben rechts in der Ecke das Diagramm bequem als Graphik herunterladen. Man muß da keinen Screenshot machen.
